templates/docker-daemon.json.erb

```diff
@@ -0,0 +1,4 @@
+{
+  <%= if not @registry_mirrors.empty? then %("registry-mirrors": [ ) + @registry_mirrors.map{|m| %("#{m}")}.join(", ") + " ],\n" end -%>
+  "seccomp-profile": "/etc/docker/seccomp/default-with-personality.json"
```

Comment:
This file is going to be added in an upcoming PR, right?

Reply:
Right, I didn't mean to commit this.

Reply:
It's been rebased out.